test/perf/perf_mutation_readers.cc

```diff
@@ -399,34 +351,106 @@ class memtable {
     }
 };
 
-PERF_TEST_F(memtable, one_partition_one_row)
+class memtable_single_row : public memtable {
```

Comment:
Why inherit from memtable, if we also have a memtable member?

Reply:
Ah, this is not `replica::memtable`, but the class above. Nevermind.

Reply:
yup, it's just a fixture which happens to be named `memtable`. i admit that it's a bad name. i just inherited it from the existing code. could change it in a follow-up PR.